### PR TITLE
static assets present in latest build

### LINKED ISSUE #75: Weaver build should include static assets
This will involve bringing in the static assets to the latest build (<latest version>x directory and 'latest' directory).

At present, it will contain the svg file for the logo. 
Future Scope: Planning to bring in resources like font etc.

### PR DESCRIPTION
**Pull request by rmathew1011**
resolves #75

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -56,8 +56,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "2mb",
-                  "maximumError": "5mb"
+                  "maximumWarning": "2000mb",
+                  "maximumError": "5000mb"
```
Comment (jeremythuff) [inline angular.json:60]: Lets see if we can bring this back down

File: projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts [hunk 2]
```diff
@@ -29,4 +29,17 @@ describe('WvrFooterComponent', () => {
...
   });
 
+  it('should positionSelf when window is resized', () => {
+    const spyOnResize = spyOn(component, 'positionSelf');
+    window.dispatchEvent(new Event('resize'));
+    expect(spyOnResize).toHaveBeenCalled();
+
+    const isSticky = false;
+    const newIsSticky = 420 <= (window.innerHeight - 65);
+    expect((isSticky !== newIsSticky)).toBeTruthy();
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts:39]: These assertions are not referencing the component

File: projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts [hunk 3]
```diff
@@ -29,4 +29,17 @@ describe('WvrFooterComponent', () => {
...
+
+    const anotherNewIsSticky = 350 <= (window.innerHeight - 65);
+    expect((isSticky !== anotherNewIsSticky)).toBeTruthy();
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts:42]: These assertions are not referencing the component

File: angular.json (hunk 1 above)
Comment (rmathew1011) [reply angular.json:60]: Reduced the size provided for budget

File: projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts (hunk 2 above)
Comment (rmathew1011) [reply projects/wvr-elements/src/lib/wvr-footer/wvr-footer.component.spec.ts]: assertions referencing the isSticky property is now being referenced.

**Pull request closed without merging**